tasks/connectors/edgescan/lib/edgescan_location_specifier.rb

```diff
@@ -28,9 +28,7 @@ def to_kenna_asset
 
           base["url"] = location if type == "url"
           base["hostname"] = location if type == "hostname"
-          base["ip_address"] = location if type == "ip"
-          base["ip_address"] = location.split("-").first if type == "block"
-          base["ip_address"] = location.split("/").first if type == "cidr"
+          base["ip_address"] = location if ["ip", "block", "cidr"].include?(type)
```

Comment:
Since we're now translating Edgescan hosts (and/or vulnerabilities) into Kenna assets, and not translating Edgescan assets into Kenna assets, do we still need this file?

I went through the references, and it looks like this code isn't used for anything.